_data/LoyalPotato/LoyalPotato.md

```diff
@@ -0,0 +1,7 @@
+---
+name: LoyalPotato
+institution: Instituto Politécnico de Leiria
```

Comment:
Change é as it is a non-ascii character.

Reply:
Done :)